File: 프로그래머스/0/120896. 한 번만 등장한 문자/한 번만 등장한 문자.js
function solution(s) {
    let charCount = {};
    let answer = '';
    [...s].forEach((value) => {
        if(charCount[value]){
            charCount[value] += 1;
        } else {
            charCount[value] = 1;
        }
    })
    answer = Object.keys(charCount).filter((value) => {
        return charCount[value] === 1;
    }).sort().join('');
    return answer;
}

File: 프로그래머스/0/120896. 한 번만 등장한 문자/한 번만 등장한 문자.py
def solution(s):
    temp = {}
    answer = []
    for c in s:
        if c in temp:
            temp[c] += 1
        else:
            temp[c] = 1
    for k, v in temp.items():
        if v == 1:
            answer.append(k)
    
    return ''.join(sorted(answer))